### ISSUE TITLE
Not installable by provided instaructions.

### ISSUE THREAD

**Issue by scottprovost** [balderdashy/sails-docs#569]
The instructions for installing will not work.
You instruct users to apt-get install nodejs.
Then you refuse to install because the version is to old.
Not installable by provided instruction
Sails.js Installation - Error
--------------------------------------------------------
Your npm-Version is too old:
Sails require npm >= 1.4.0 (you currently have 1.3.10)

Please update the installed npm (Node Package Manager)
 to install Sails.

Many users removed current versions of Node to follow your instructions and apt-get install.
Only to find they have permanently damaged their systems and can never run any Nodejs software.
Removing the symbolic link to nowhere created buy nodejs legacy can get them back to usability.

**Comment by sailsbot** [comment]
Thanks for posting, @scottprovost.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
